django_server/api_keys/FEATURES.md

```
@@ -0,0 +1,161 @@
+- **HTTPS Only**: Always use encrypted connections
+- **Rate Limiting**: Implement request throttling
+- **Monitoring**: Track unusual API usage patterns
+
+## Event Tracking
+- api_key_generated
+- api_key_deleted
+- browser_extension_key_generated
+- browser_extension_key_deleted
+- invite_generated
+- invite_claimed
+- invite_revoked
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)